.changes/1.4.2.md

```diff
@@ -17,7 +17,7 @@
 - Update `snowflake-connector-python` to 3.0 to address CVE-2022-42965 ([#476](https://github.com/dbt-labs/dbt-snowflake/pull/476))
 
 ### Contributors
-- [@dataders](https://github.com/dataders) ([#331](https://github.com/dbt-labs/dbt-spark/issues/331))
+- [@dataders](https://github.com/dataders) ([#331](https://github.com/dbt-labs/dbt-snowflake/issues/331))
 - [@dbeatty10](https://github.com/dbeatty10) ([#476](https://github.com/dbt-labs/dbt-snowflake/pull/476))
-- [@joshuataylor](https://github.com/joshuataylor) ([#331](https://github.com/dbt-labs/dbt-spark/issues/331))
-- [@mikealfare](https://github.com/mikealfare) ([#331](https://github.com/dbt-labs/dbt-spark/issues/331), [#476](https://github.com/dbt-labs/dbt-snowflake/pull/476), [#476](https://github.com/dbt-labs/dbt-snowflake/pull/476))
+- [@joshuataylor](https://github.com/joshuataylor) ([#331](https://github.com/dbt-labs/dbt-snowflake/issues/331))
+- [@mikealfare](https://github.com/mikealfare) ([#331](https://github.com/dbt-labs/dbt-snowflake/issues/331), [#476](https://github.com/dbt-labs/dbt-snowflake/pull/476), [#476](https://github.com/dbt-labs/dbt-snowflake/pull/476))
```

Comment:
Mike and Anders shouldn't be included here really.  Not a huge deal though.  But while you're updating changie you should update the core team list in `.changie.yaml` on line 83 to include the current team.  That way future 1.4 releases will exclude correctly.  Larger lift is implementing the auto pull from GitHUb teams we use in later version but it's not worth the time in my opinion.